pyproject.toml

```diff
@@ -14,6 +14,7 @@ classifiers = [
     "License :: OSI Approved :: MIT License",
     "Operating System :: POSIX",
 ]
+dependencies = ["libbash"]
```

Comment:
I agree with your intuition — although I can’t think of an obvious scenario that would cause a problem. The code right now only uses libbash in a runtime capacity for a handful of enums, so I can remove the dependency after copying those over.